src/service.ts

```diff
@@ -182,6 +183,23 @@ export class DebugService implements IDebugger {
     return reply.body.scopes;
   };
 
+  getVariable = async (_: any, variable: DebugProtocol.Variable) => {
+    if (!variable && variable.variablesReference !== 0) {
```

Comment:
Can the 2nd part of this expression be evaluated?

Reply:
Some additional variable like in Class A (from gif) foo or bar have variable reference equal 0.
I guess that means we  cant get more description of this variable

Reply:
By that I mean:

- can `variable` be null or undefined?
- if so then isn't `variable.variablesReference` going to break?

Reply:
I will check that